scripts/haproxy/s3haproxyconfig/s3haproxyconfig/s3_haproxy_config.py

```diff
@@ -42,11 +45,20 @@ def run():
     machine_id = mcid_file.read().strip()
     mcid_file.close()
 
-    #Get necessary info from s3conf_store
+    #Get necessary info from confstore
     localhost = '127.0.0.1'
-    pvt_ip = s3conf_store.get_privateip(machine_id)
-    pub_ip = s3conf_store.get_publicip(machine_id)
-    numS3Instances = int(s3conf_store.get_s3instance_count(machine_id))
+    pvt_ip = provisioner_confstore.get_config(
+      local_confstore.get_config(
+        'CONFSTORE_PRIVATE_FQDN_KEY').format(machine_id))
+
+    pub_ip = provisioner_confstore.get_config(
+      local_confstore.get_config(
+        'CONFSTORE_PUBLIC_FQDN_KEY').format(machine_id))
+
+    numS3Instances = int(provisioner_confstore.get_config(
```

Comment:
Try common property

Reply:
@property decorator in python is used for getter-setter functions. I read it here: https://www.programiz.com/python-programming/property

But I have added new functions to encapsulate the above code.